### PR TITLE
Initial pass at OLVM doc

### PR DESCRIPTION
**Pull request by pfmackin**
This is the initial pass at OLVM documentation.  This document explains how to create a Kubernetes cluster on an OLVM instance.  This is initially for a developer release.

### CODE REVIEW

File: doc/cluster-management/olvm.md [hunk 1]
```diff
@@ -0,0 +1,546 @@
+**NOTE: This is a developer release**
+
+# OLVM Cluster API Provider
+The OLVM Cluster API Provider allows you to create Kubernetes clusters on an
```
Comment (dkrasins) [inline doc/cluster-management/olvm.md:4]: The first instance of an acronym should be spelled out.  "Oracle Linux Virtualization Manager (OLVM)".

Comment (pfmackin) [reply doc/cluster-management/olvm.md:4]: yeah i thought i did that, will fix

**Review by dkrasins — DISMISSED** [review]

File: doc/cluster-management/olvm.md [hunk 2]
```diff
@@ -0,0 +1,546 @@
+**NOTE: This is a developer release**
+
+# OLVM Cluster API Provider
+The Oracle Linux Virtualization Manager (OLVM) Cluster API Provider allows you to create Kubernetes clusters on an
+existing OLVM instance. The cluster nodes can be spread
+across multiple OLVM hosts, where both the control plane and worker nodes can
+be scaled in and out as desired. Using the Oracle Cloud Native Environment CLI (`ocne`),
```
Comment (mgianatagh) [inline doc/cluster-management/olvm.md:7]: Is it ok to use `ocne`?  I thought we were not supposed to.

Comment (pfmackin) [reply doc/cluster-management/olvm.md:7]: that is the CLI, i define it up above.

File: doc/cluster-management/olvm.md [hunk 3]
```diff
@@ -0,0 +1,546 @@
...
+INFO[2024-12-20T13:46:56-05:00] Uploading boot image to pod ocne-system/ocne-image-builder: ok 
+INFO[2024-12-20T13:47:41-05:00] Downloading boot image from pod ocne-system/ocne-image-builder: ok 
+INFO[2024-12-20T13:47:41-05:00] New boot image was created successfully at /Users/pmackin/.ocne/images/boot.qcow2-1.30-amd64.olvm 
+```
+
+## Uploading the OLVM OCK image
+Upload the OCK image that you just created to your 
+
+```
+ocne image upload --type olvm --arch amd64 --file  /Users/pmackin/.ocne/images/boot.qcow2-1.30-amd64.olvm   --config /Users/pmackin/.ocne/olvm-demo-cluster-config.yaml --kubeconfig $KUBECONFIG
```
Comment (mgianatagh) [inline doc/cluster-management/olvm.md:348]: Your name is in the example

Comment (pfmackin) [reply doc/cluster-management/olvm.md]: fixed

**Review by mgianatagh — APPROVED** [review]

**Review by dkrasins — APPROVED** [review]

**Pull request merged**